src/main/offline/io/stargate/sgv2/jsonapi/service/cqldriver/sstablewriter/OfflineCommandsProcessor.java

```diff
@@ -163,7 +163,11 @@ public Triple<BeginOfflineSessionResponse, CommandContext, String> beginSession(
     return new ImmutableTriple<>(
         beginOfflineSessionResponse,
         commandContext,
-        beginOfflineSessionCommand.getFileWriterParams().createTableCQL());
+        new SchemaInfo(
```

Comment:
As part of the table creation, we should be creating the indexes required also before getting the data loaded via the sideloader so that the indexing is done while loading.